Libplanet.Explorer/Store/MySQLRichStore.cs

```diff
@@ -360,33 +600,178 @@ public void StoreSignerReferences(TxId txId, long txNonce, Address signer)
             int limit = int.MaxValue)
         {
             using QueryFactory db = OpenDB();
-            var query = db.Query("updated_address_references")
+            var query = db.Query(UpdatedAddressRefDbName)
                 .Where("updated_address", updatedAddress.ToByteArray())
                 .Offset(offset)
                 .Limit(limit)
                 .Select("tx_id");
             query = desc ? query.OrderByDesc("tx_nonce") : query.OrderBy("tx_nonce");
             return query.OrderBy("tx_nonce")
-                .Get<byte[]>()
-                .Select(bytes => new TxId(bytes));
+                .Get<string>()
+                .Select(txString => new TxId(ByteUtil.ParseHex(txString)));
+        }
+
+        public void LoadDatabase()
+        {
+            // flush streamwriter
+            blockBulkWriter.Flush();
+            txBulkWriter.Flush();
+            txRefBulkWriter.Flush();
+            signerRefBulkWriter.Flush();
+            updatedAddressRefBulkWriter.Flush();
+
+            // bulk load files to database
+            BulkLoad(TxDbName, txFilePath);
+            BulkLoad(TxRefDbName, txRefFilePath);
+            BulkLoad(SignerRefDbName, signerRefFilePath);
+            BulkLoad(
+                UpdatedAddressRefDbName,
+                updatedAddressRefFilePath);
+            BulkLoad(BlockDbName, blockFilePath);
+
+            // delete loaded files
+            File.Delete(blockFilePath);
+            File.Delete(txFilePath);
+            File.Delete(txRefFilePath);
+            File.Delete(signerRefFilePath);
+            File.Delete(updatedAddressRefFilePath);
+
+            // create new files
+            blockFilePath = Path.GetTempPath()
+                + Path.GetRandomFileName() + BlockDbName + ".tmp";
+            txFilePath = Path.GetTempPath()
+                + Path.GetRandomFileName() + TxDbName + ".tmp";
+            txRefFilePath = Path.GetTempPath()
+                + Path.GetRandomFileName() + TxRefDbName + ".tmp";
+            signerRefFilePath = Path.GetTempPath()
+                + Path.GetRandomFileName() + SignerRefDbName + ".tmp";
+            updatedAddressRefFilePath = Path.GetTempPath()
+                + Path.GetRandomFileName() + UpdatedAddressRefDbName + ".tmp";
+
+            // dispose flushed streamwriter and reset
+            blockBulkWriter.Dispose();
+            txBulkWriter.Dispose();
+            txRefBulkWriter.Dispose();
+            signerRefBulkWriter.Dispose();
+            updatedAddressRefBulkWriter.Dispose();
+            blockBulkWriter = new StreamWriter(blockFilePath);
+            txBulkWriter = new StreamWriter(txFilePath);
+            txRefBulkWriter = new StreamWriter(txRefFilePath);
+            signerRefBulkWriter = new StreamWriter(signerRefFilePath);
+            updatedAddressRefBulkWriter = new StreamWriter(
+                updatedAddressRefFilePath);
+        }
```

Comment:
@planetarium/9c-backend 
This gets the job done (`flush streamwriter` => `load files to db` => `delete files` => `create new files` => `reset streamwriter`)... but I feel like the code is way too ugly(?). I'm open for any suggestions programmatically and/or style-wise :pray: